pkg/virtctl/imageupload/imageupload.go

```diff
@@ -210,7 +211,7 @@ func (c *command) run(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	defer file.Close()
+	defer util.CloseIOAndCheckErr(file, nil)
```

Comment:
Why do we choose to ignore the error here?

Reply:
The file is opened in RO mode. In this case we only log the error message.